src/injectors.js

```diff
@@ -4,13 +4,19 @@ export const head = function(w, d, t, o, r, c, s) {
   (c = d.createElement(t)), (s = d.getElementsByTagName(t)[0]);
   c.appendChild(d.createTextNode(o.text));
   c.onload = r(o);
+  if (o.nonce !== undefined && o.nonce !== null) {
```

Comment:
just do `if(!o.nonce)`, nonce as just an empty string is an invalid nonce anyway.